feature-libs/cart/main/components/cart/added-to-cart-dialog/added-to-cart-dialog-event.listener.spec.ts

```diff
@@ -75,4 +98,14 @@ describe('AddToCartDialogEventListener', () => {
       );
     });
   });
+
+  describe('closeModal', () => {
+    it('Should close the add to cart dialog', () => {
```

Comment:
perhaps we can also test what happens when the active modal ref is not equal to the listener modal ref.  It might help the unit test branch coverge %